File: Unnati-Sadh/q3.cpp
//Function to remove duplicates from sorted linked list.
Node* removeDuplicates(Node *head)
{
  
   struct Node* current = head;
    
    //traversing through the linked list.
    while (current->next != NULL)       
    {
        
        if (current->data == current->next->data)
           current->next = current->next->next;
           
        
        else
           current = current->next; 
    }
    return head;
    
}

File: Akshit-Parmar/q3.py
def removeDuplicates(head):
    
    temp = head
    if temp is None:
        return
    while temp.next is not None:
        if temp.data == temp.next.data:
                new = temp.next.next
                temp.next = None
                temp.next = new
        else:
                temp = temp.next
    return head
